sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/javaCode.scala

```diff
@@ -155,6 +170,17 @@ object Block {
 
   val CODE_BLOCK_BUFFER_LENGTH: Int = 512
 
+  /**
+   * A custom string interpolator which inlines all types of input arguments into a string without
```

Comment:
nit: maybe this comment is better to be put to the `InlineBlock` class, do you agree?

Reply:
I think part of this comment can be moved to `InlineBlock`.

Reply:
I will also do some improvement of document.